qml/ConnectionMenu.qml

```diff
@@ -205,9 +192,29 @@ Item {
                         bottom: 1
                         top: 65535
                     }
-                    onEditingFinished: {
-                        connect(AbstractLinkNamespace.Udp, udpIp.text, text)
+                }
+            }
+
+            Button {
+                text: "Connect"
+                Layout.fillWidth: true
```

Comment:
Can we do something like this to block connecting to invalid addreses?
```suggestion
                Layout.fillWidth: true
                // Disable if using an invalid UDP address
                enabled: conntype.currentIndex === 0 || udpIp.isValid
```
Can we get something like this to forbid connections to bad addresses?